tests/phpunit/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -255,7 +255,6 @@ public function get_upe_available_payment_methods_provider() {
 					WC_Stripe_UPE_Payment_Method_Alipay::STRIPE_ID,
 					WC_Stripe_UPE_Payment_Method_Giropay::STRIPE_ID,
 					WC_Stripe_UPE_Payment_Method_Klarna::STRIPE_ID,
-					WC_Stripe_UPE_Payment_Method_Affirm::STRIPE_ID,
```

Comment:
Because Affirm is only available to US and CA merchants, I need to remove it from this 
"Non-US" test.